TabLayout/tests/TabGroup-specs.js

```diff
@@ -15,15 +18,14 @@ describe('TabGroup specs', () => {
 				]}
 			/>
 		);
+		const homeIconElement = screen.getByTestId('tabGroup').children.item(0).children.item(1).children.item(0).textContent;
+		const homeIcon = screen.getByText(homeIconElement);
 
-		const expected = 1;
-		const actual = subject.find('Tab').length;
-
-		expect(actual).toEqual(expected);
+		expect(homeIcon).toBeInTheDocument();
```

Comment:
this looks like a truism. I recommend to compare the DOM when all tabs have an icon and when one tab doesn;t have. I see a difference: this node: `          <div
            aria-multiselectable="false"
            class="tabs"
            data-spotlight-container="true"
            data-spotlight-container-disabled="false"
            data-spotlight-container-muted="false"
            data-spotlight-id="container-1"
            orientation="vertical"
            role="group"
          >`